scripts/pipeline/publish-docker-images.sh

```diff
@@ -6,7 +6,7 @@ if [ -n "$GITHUB_HEAD_REF" ]; then
   export GITHUB_BRANCH="$GITHUB_HEAD_REF"
 fi
 if [[ "$GITHUB_EVENT_NAME" = "release" ]]; then
```

Comment:
i suggest to reassign the TAG env var after init the TAG env var (line 11) instead of the GITHUB_BRANCH since the value isn't a GITHUB_BRANCH

export TAG=trubudget/"$PROJECT_NAME":"$SOURCE_TAG"